README.md

```diff
@@ -142,7 +150,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is the blueprint of how the database is constructed. One purpose of a schema is to specify how the facts that are inputted into the database apply to the end users. A one-to-many relationship is diagrammed in a SQL database by having two separate (rectangular) entities connected by one (diamond) relationship in a 1:N format. The '1' represents the one instance associated with the relationship, and the N represents more than one instance associated with a relationship. In this case, people can have more than one wish, but wishes cannot have more than one person. Therefore, 1 would be on the people side and wishes would be on the N side.
```

Comment:
I believe you are describing the ERD from the previous question.  The schema is a manifest of the tables and columns contained in the database.  We were expecting to see examples of tables and columns.